### PR TITLE
Add blue arrow option to action link

### PR DESCRIPTION
**Pull request by andysellick**
## What
Adds an option to put a blue arrow on the action link.

## Why
Because there aren't enough options on this component already. No, actually it's because we want this for some redesign changes on the coronavirus page.

## Visual Changes

Screenshot 2021-09-30 at 17 01 10

Trello card: https://trello.com/c/KeJVLjie/649-collections-frontend-implementation-of-the-new-coronavirus-header-design

### CODE REVIEW

File: app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss [hunk 1]
```diff
@@ -92,6 +92,18 @@
   }
 }
 
+.gem-c-action-link--blue-arrow {
+  &:before {
+    width: 35px;
+    height: 30px;
+    background: image-url("govuk_publishing_components/action-link-arrow--blue.png");
```
Comment (danacotoran) [inline app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss:99]: I think you forgot to add the PNG to `assets/images`.

**Comment by danacotoran** [comment]
It's not within the scope of this PR but I noticed some of the variants basically have the same styling with the only different thing being the image used as the arrow. I wonder if we can consolidate that using SASS mixins to make the component stylesheet less repetitious.

File: app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss (hunk 1 above)
Comment (andysellick) [reply app/assets/stylesheets/govuk_publishing_components/components/_action-link.scss:99]: 🤦

**Review by danacotoran — APPROVED** [review]
➡️ 🚀 !

**Pull request merged**